README.md

```diff
@@ -146,6 +146,16 @@ There are a lot of ways to do this, including [SAML and forward authentication](
 Grist has been tested with [Authentik](https://goauthentik.io/), [Auth0](https://auth0.com/),
 and Google/Microsoft sign-ins via [Dex](https://dexidp.io/).
 
+## Translations
+
+We use [Weblate](https://hosted.weblate.org/engage/grist/) to manage translations.
+Thanks to everyone who is pitching in. Thanks especially to the ANCT developers who
+did the hard work of making a good chunk of the application localizable. Merci bien!
+
+<a href="https://hosted.weblate.org/engage/grist/">
+<img src="https://hosted.weblate.org/widgets/grist/-/open-graph.png" alt="Translation status" />
```

Comment:
I suggest adding a style to limit the width so this image doesn't take up so much space:
```suggestion
<img src="https://hosted.weblate.org/widgets/grist/-/open-graph.png" alt="Translation status" width=480 />
```